utils/contributors.js

```diff
@@ -59,6 +59,33 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+
```

Comment:
LGTM but would suggest a change:
- Remove this extra blank line as it can cause merge conflicts.